resources/views/cooperation/admin/super-admin/cooperations/users/show.blade.php

```diff
@@ -94,7 +96,7 @@
                         }
                     }).done(function () {
                         // just reload the page
-                        location.reload();
+                        // location.reload();
```

Comment:
Is this correct ?

Reply:
Yes. It's unnecessary. Doesn't do it on remove either.